### ISSUE TITLE
security concern

### ISSUE THREAD

**Issue by abumalick** [srisankethu/StickersStore#1]
Hello,

I am not an express expert but when I read your code it seems to me that all your users will share the same session.

https://github.com/srisankethu/StickersStore/blob/master/app.js#L7

You can test what I say by logging in with a browser, and visit the site with another browser. You should have the same session...

**Comment by srisankethu** [comment]
Yes yes I know. I haven't added sessions. I have added that to my TODO list. Thanks for checking my code.
